config/locales/en.yml

```diff
@@ -1488,7 +1488,7 @@ en:
         signup: "signup"
         contact: "contact"
         require_customer_login: "Only approved customers can access this shop."
-        require_login_html: "If you're already an approved customer, %{login} or %{signup} to proceed."
+        require_login_html: "If you're already an approved customer, %{login} to proceed."
```

Comment:
I'm wondering if there is no need to rename this key? Will all the instance managers / i18n managers be aware that the key has changed and the `%{signup}` placeholder should be removed?
Because if the `%{signup}` placeholder stays in the key, I guess it will _break_ the page.

Reply:
Yes, keeping the %{signup} placeholder seems to break the page with this error:

```
     ActionView::Template::Error:
       missing interpolation argument :signup in "If you're already an approved customer, %{login} or %{signup} to proceed." ({:login=>"<a auth=\"login\" data-action=\"click->login-modal#call\">login</a>"} given)
 
```

Reply:
So, if keeping the placeholder in the i18n string breaks the page, I think we should create another i18n key: therefore the i18n managers will be notified and the page will never break.

Reply:
Yeap, that's exactly what I meant ;) 
I'd squash those two commits, and I think it's all good now!